lib/libevent.c

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
It appears that the rest of the codebase here uses tabs.  Please reformat to use the same style as the rest of the code (not doing so reduces the chance it will get merged into upstream).